### PR TITLE
Add es-MX language support

### PR DESCRIPTION
**Pull request by Eencsio**
### Context

Add new language support for es-MX

### How has this been tested?

add e2e test for spanish support

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Related issue(s):
1.  4696 https://github.com/handsontable/handsontable/issues/4696

### Checklist:

- [X] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by AMBudnik** [comment]
Thank you for your contribution @Eencsio 

I can see that you have also signed CLA so I have refreshed the tests and created an issue for the community votes at https://github.com/handsontable/handsontable/issues/4900

**Comment by AMBudnik** [comment]
We got all needed confirmations. I am changing the label.

File: src/i18n/languages/es-MX.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Handsoncode
```
Comment (wszymanski) [inline src/i18n/languages/es-MX.js]: Please change from `Handsoncode` to your name or leave it empty.

**Comment by wojciechczerniak** [comment]
Hi @Eencsio, can you check the requested changes from the reviewer? I would love to see this merged and published with our upcoming release this month 🎉

**Comment by Eencsio** [comment]
HI! I've updated my commit to only merge my language file, can someone help me to resolve the merging conflict??, I think that I can resolve it because of the write access permissions

**Comment by Eencsio** [comment]
All done! check if now is right,:D

**Comment by wojciechczerniak** [comment]
Great! Thank you. Your translation will be released with Handsontable next week.

**Comment by adrianszymanski89** [comment]
Hi @Nciso. We just announced an introduction of a new functionality called `Copy with headers`. Could you help us to translate three short lines for the context menu?

Here you can find more information about it: https://github.com/handsontable/handsontable/issues/10112

**Comment by adrianszymanski89** [comment]
@Nciso Yes, it's still being developed. Thank you very much for such a quick response!

**Pull request merged**